src/js/view/view.js

```diff
@@ -379,7 +379,9 @@ define([
 
         function _setAspectRatio(){
             var styleTarget = '#'+_playerElement.id+'.jw-aspect-mode:before';
-            document.styleSheets[0].addRule(styleTarget, 'padding-top: ' + _model.aspectratio + '%');
+            var targetStylesheet = document.styleSheets[0];
```

Comment:
You should not be modifying an arbitrary style sheet on the page. Query the element (if you don't already have a reference to it) and set inline styles.